Content.Client/_CD/Records/UI/RecordEntryViewPopup.xaml.cs

```diff
@@ -1,30 +1,39 @@
 using Content.Client.UserInterface.Controls;
 using Content.Shared._CD.Records;
 using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
 using Robust.Client.UserInterface.RichText;
 using Robust.Client.UserInterface.XAML;
 using Robust.Shared.Utility;
 
 namespace Content.Client._CD.Records.UI;
 
-// TODO: Add a scrollbar to this. I tried for over an hour and I could not get it to work in a way that I liked.
-
 [GenerateTypedNameReferences]
 public sealed partial class RecordEntryViewPopup : FancyWindow
 {
     // We don't permit headings because it looks ugly
     private static readonly Type[] AllowedTags =
-    {
+    [
         typeof(BoldItalicTag),
         typeof(BoldTag),
         typeof(BulletTag),
         typeof(ColorTag),
-        typeof(ItalicTag)
-    };
+        typeof(ItalicTag),
+    ];
```

Comment:
Does permitting `[head]` tags look alright? If so, we should permit them. The only reason why they were not included before was because of my bad UI design.

Reply:
It might be fine? Your call

<details><summary>Click me!</summary>
<br>

![image](https://github.com/user-attachments/assets/0b0b7681-8ea2-4360-9e5c-dfccb91e5c3a)

![image](https://github.com/user-attachments/assets/f334f6b6-dab7-4429-bc6d-81a214e31c43)

</details>

Reply:
Yep, I think that looks okay. I think you should be able to get rid of the whitelist entirely.